cardano-api/src/Cardano/Api/SerialiseCBOR.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE DefaultSignatures #-}
+
+-- | CBOR serialisation
+--
+module Cardano.Api.SerialiseCBOR
```

Comment:
Ah, I might prefer `SerialiseCbor` or `Serialise.Cbor`, but we've already named the type `SerialiseAsCBOR` so this is fine.

Reply:
We can rename the modules later if we want to. They're all internal, not exposed.